pypeapp/pypeLauncher.py

```diff
@@ -40,17 +38,26 @@ def print_info(self):
             infos.append(("Using Muster at",
                           os.environ.get("MUSTER_REST_URL")))
 
-        if host:
-            infos.append(("Logging to mongodb", "{}/{}".format(
-                host, database)))
-            if port:
-                infos.append(("  - port", port))
-            if username:
-                infos.append(("  - user", username))
-            if collection:
-                infos.append(("  - collection", collection))
-            if auth_db:
-                infos.append(("  - auth source", auth_db))
+        if components["host"]:
+            if os.environ.get("PYPE_LOG_MONGO_COL"):
```

Comment:
These prints and conditions does not make much sence. `Logging to mongodb` should be always printed if host is is components. And environments should not be used here at all (because are optional)...

First line should be only host. Database name should be printed as line before collection. Both can be imported as `pypeapp.lib.log.COLLECTION` and `pypeapp.lib.log.DATABASE` to get really used values.